### PR TITLE
Added the calender screen 

### PR DESCRIPTION
**Pull request by imadhb**
This PR adds the calendar screen with day highlights based on mood. More functionality will be added.
image

### CODE REVIEW

File: VitalTrackApp/App.js [hunk 1]
```diff
@@ -31,6 +32,7 @@ function AppNavigator() {
       </Stack.Screen>
       <Stack.Screen name="Settings" component={SettingsScreen} />
       <Stack.Screen name="Entries" component={EntriesScreen} />
+       <Stack.Screen name="Calendar" component={CalendarScreen} options={{ headerShown: false }} />
```
Comment (MerwinAamir1) [inline VitalTrackApp/App.js:35]: why are we hiding the header for the calendar but not everything else?

**Review by MerwinAamir1 — CHANGES_REQUESTED** [review]

File: VitalTrackApp/App.js (hunk 1 above)
Comment (imadhb) [reply VitalTrackApp/App.js:35]: I'll put it back i was having issues with the CSS for a calendar so I omitted  out to try to fit the calendar better

Comment (MerwinAamir1) [reply VitalTrackApp/App.js:35]: you can either take it out or omit the header for every other component

**Comment by MerwinAamir1** [comment]
but this looks really good!

File: VitalTrackApp/App.js (hunk 1 above)
Comment (imadhb) [reply VitalTrackApp/App.js:35]: I hid the header for all screens

**Review by MerwinAamir1 — APPROVED** [review]

**Comment by MerwinAamir1** [comment]
Make sure to squash and merge this

**Pull request merged**